features/u.yml

```
@@ -1,7 +1,6 @@
-draft_date: 2024-06-05
 name: <u>
-description: The `<u>` element TODO.
+description: The `<u>` HTML element underlines text and other content.
```

Optionally, we could align all of these to use the same pattern, like this:

```suggestion
description: The `<u>` HTML element makes non-textual annotations, styling text as underlined by default.
```